src/lib/utils/SynapseClient.tsx

```diff
@@ -9,18 +9,25 @@ import { FaceFacetColumnValuesRequest } from './jsonResponses/Table/FacetColumnR
 
 // TODO: Create JSON response types for return types
 const DEFAULT_ENDPOINT = 'https://repo-prod.prod.sagebase.org/'
+const DEFAULT_SWC_ENDPOINT = 'https://www.synapse.org/'
 
 function delay(t: any) {
   return new Promise((resolve) => {
     setTimeout(resolve.bind(null, null), t)
   })
 }
+function parseJSON(response: any) {
+  return response.text()
+  .then((text: string) => {
+    return text ? JSON.parse(text) : {}
+  })
+}
 const fetchWithExponentialTimeout = (url: string, options: any, delayMs: any, retries: number): Promise<any> => {
   return fetch(url, options)
     .then((resp) => {
       if (resp.status > 199 && resp.status < 300) {
         // ok!
-        return resp.json()
+        return parseJSON(resp)
```

Comment:
Should all returns from this function have this same format? [e.g. this line below](https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/76/files?utf8=%E2%9C%93&diff=split#diff-568e1d888d041ef55c0cd01da75ef9c7R45)

Reply:
probably